lib/tty/box.rb

```diff
@@ -187,10 +188,18 @@ def error(message, **opts)
     #   the styling for the front and background
     #
     # @api public
-    def frame(*content, top: nil, left: nil, width: nil, height: nil, align: :left,
-              padding: 0, title: {}, border: :light, style: {})
+    def frame(*content,
+              align: :left,
+              border: :light,
+              height: nil,
+              left: nil,
+              padding: 0,
+              style: {},
+              title: {},
+              top: nil,
+              width: nil
+      )
```

Comment:
My preference would be to keep it compact as it were for now. Though I don't have strong opinions on this. Maybe I would need to warm up to this style of keywords listing...